### DISCUSSION TITLE
Possible to define ZL level?

### DISCUSSION THREAD

**Discussion by stuieH** [kubilus1/autoortho#492] [Ideas]
In order to save cache size and steam time would it be possible to define a ZL level to download e.g. ZL16 to ZL18?  Even better would be to automatically download lower ZL levels the higher in altitude you are at. E.g. if below 5000 ft use ZL18 then reduce as one gains altitude in say 5000 feet increments until above FL200 when downloads would limit to say ZL14.

Also, does the current version roll the cache over once full? 

Thanks

**Comment by stuieH** [comment]
Having tried the program earlier today, I noticed that the ZL tiles downloaded ranged from ZL18 at the airport down to ZL16 as one moved away from the airport surround.  So it does seem the program is adjusting the ZL level as one flies away from the ground.  It would be nice to know the minimum ZL level that the program will use.

cheers.

**Comment by Jeff2122** [comment]
ZL12 is the default. I use ZL14 but I don't notice any gain in quality. 
AO v0.7.2
min_zoom = 14
maxwait = 5.0
fetch_threads 32
WinFSP = true
cache = 20GB

**Reply by stuieH** [reply]
Thanks Jeff,
Didn’t realise one could change the min ZL in a file.  Where does one find that settings file?

**Reply by Jeff2122** [reply]
It is in your user's folder at 

c:\users\<YOURNAME>\autoortho.

The file has no extension. "autoortho." is the file you need to edit.
